db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_200_207_200_038) do
+ActiveRecord::Schema.define(version: 20_200_213_210_949) do
```

Comment:
Seems like schema changes are not from this PR... Although we should update the others to match the state of the database (- limitations pointed by @cdebourcy) we should not include them... means manually reverting the remaining changes...